tyk-pro/templates/statefulset-enterprise-portal.yaml

```diff
@@ -152,7 +152,7 @@ spec:
         livenessProbe:
           httpGet:
             scheme: "HTTP{{ if .Values.enterprisePortal.tls }}S{{ end }}"
-            path: /
+            path: /live
```

Comment:
Is this change backward compatible? What happens if users use 1.x.y which had no support for `/live` path? Should we make path field configurable?

Reply:
That make sense. Will update PR accordingly.

Reply:
@buraksekili Probes are added in portal v1.2.0 https://tyk.io/docs/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.2.0/

Reply:
thank you for the reference 🙏